### PR TITLE
Dartdoc warnings

### PR DESCRIPTION
**Pull request by goderbauer**
Follow-up to https://github.com/flutter/flutter/pull/136975.

### CODE REVIEW

File: packages/flutter/lib/foundation.dart [hunk 1]
```diff
@@ -47,3 +47,16 @@ export 'src/foundation/stack_frame.dart';
 export 'src/foundation/synchronous_future.dart';
 export 'src/foundation/timeline.dart';
 export 'src/foundation/unicode.dart';
+
```
Comment (goderbauer) [inline packages/flutter/lib/foundation.dart:50]: The code below produces these warnings:

```
dartdoc:stderr:   error: generic type handled as HTML: """This is a `List<int>`."""
dartdoc:stderr:     from foundation.bar: (file:///b/s/w/ir/x/w/%C3%81%20flutter%20sdk/packages/flutter/lib/foundation.dart:55:14)
dartdoc:stderr:   error: generic type handled as HTML: """List<int> foo = [];"""
dartdoc:stderr:     from foundation.baz: (file:///b/s/w/ir/x/w/%C3%81%20flutter%20sdk/packages/flutter/lib/foundation.dart:62:14)
dartdoc:stderr:   error: generic type handled as HTML: """This is a List<int>."""
dartdoc:stderr:     from foundation.foo: (file:///b/s/w/ir/x/w/%C3%81%20flutter%20sdk/packages/flutter/lib/foundation.dart:52:14)
```
@srawlins Is this working as intended? I would have expected to only get a warning on line 52.

Comment (srawlins) [reply packages/flutter/lib/foundation.dart:50]: Oof no that warning is definitely super broken.

Comment (goderbauer) [reply packages/flutter/lib/foundation.dart:50]: https://github.com/dart-lang/dartdoc/issues/3545

**Review by gspencergoog — APPROVED** [review]
32384589-a60f0e74-c078-11e7-9bc1-e5b5287aea9d

**Review by srawlins — APPROVED** [review]

**Pull request merged**